EM beta: make adding an app as resource more discoverable (#15752)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/accesspackageresource.md

| Property     | Type        | Description |
|:-------------|:------------|:------------|
|accessPackageResourceEnvironment|[accessPackageResourceEnvironment](../resources/accesspackageresourceenvironment.md)|Contains the environment information for the resource. This can be set using either the `@odata.bind` annotation or the environment's *originId*.|

> |attributes|[accessPackageResourceAttribute](../resources/accesspackageresourceattribute.md) collection| Contains attribute information for the attributes that are required to be collected from the requestor and provided to the resource application.|

Reviewer: ```suggestion
|attributes|[accessPackageResourceAttribute](../resources/accesspackageresourceattribute.md) collection| Contains information about the attributes to be collected from the requestor and sent to the resource application. |
```
